bank-sdk/component-api-example-app/src/main/java/net/gini/pay/appcomponentapi/digitalinvoice/LineItemDetailsExampleActivity.kt

```diff
@@ -113,7 +113,7 @@ class LineItemDetailsContract : ActivityResultContract<LineItemDetailsInput, Sel
 
     companion object {
         private const val EXTRA_IN_SELECTABLE_LINE_ITEM = "EXTRA_IN_SELECTABLE_LINE_ITEM"
-        private const val EXTRA_IN_RETURN_REASONS = "EXTRA_IN_RETURN_REASONS"
+        private const val EXTRA_IN_RETURN_REASONS = "EMainActivityXTRA_IN_RETURN_REASONS"
```

Comment:
Just to double-check is this a change you want to apply to 🙈?

Reply:
Oh, thank you, thank you! How did that happen?! 🙈